tests/unit/test_middleware_security.py

```
@@ -0,0 +1,399 @@
+
+# ---------------------------------------------------------------------------
+# Helpers
+# ---------------------------------------------------------------------------
+
+
+def _bare_app() -> FastAPI:
+    """FastAPI app with a single GET /test route and no middleware."""
+    app = FastAPI()
+
+    @app.get("/test")
+    async def test_endpoint():
```

Ruff (0.15.13)

[warning] Missing return type annotation for private function `test_endpoint`

(ANN202)